BEGINNERS.md

```diff
@@ -0,0 +1,291 @@
+# Minimal Snowdrift set up instructions
+
+This is a small tutorial on contributing to Snowdrift. You don't have to
+be tech savvy, but it helps. This is a minimal-ish description of what
+you need to do to get Snowdrift running locally and to start making and
+submitting updates.
+
+Although we recommend deeper study to understand how all of this works,
+you *can* simply follow the steps below, and it will work well enough
+to get started.
+
+Advanced users should adapt any of these instructions as seen fit.
+
+These instructions assume you use Debian testing or some variant of Ubuntu.
+Our general [GUIDE](GUIDE.md) has some notes for other systems.
+
+If you need *any* help at any point, come say "hi" at our freenode.net IRC
+channel [#snowdrift](https://snowdrift.coop/p/snowdrift/w/irc).
+We are always happy to assist and answer *any* questions!
+
+## Installing
+
+Open up a terminal, and run these commands, exactly as you see them. You
+could even copy and paste all of them into your terminal at once, and it
+would work.
+
+    sudo aptitude update
+    sudo aptitude install curl git postgresql postgresql-client
+    sudo ln -s /usr/share/x86_64-linux-gnu/libgmp.so.10 /usr/share/x86_64-linux-gnu/libgmp.so
```

Comment:
Agreed on both counts. I recommend just adding libgmp-dev to the list
of installed packages.

On Mon, Apr 20, 2015 at 11:16:23AM -0700, Stephen Paul Weber wrote:
> In BEGINNERS.md:
> 
> > +These instructions assume you use Debian testing or some variant of Ubuntu.
> > +Our general [GUIDE](GUIDE.md) has some notes for other systems.
> > +
> > +If you need *any* help at any point, come say "hi" at our freenode.net IRC
> > +channel [#snowdrift](https://snowdrift.coop/p/snowdrift/w/irc).
> > +We are always happy to assist and answer *any* questions!
> > +
> > +## Installing
> > +
> > +Open up a terminal, and run these commands, exactly as you see them. You
> > +could even copy and paste all of them into your terminal at once, and it
> > +would work.
> > +
> > +    sudo aptitude update
> > +    sudo aptitude install curl git postgresql postgresql-client
> > +    sudo ln -s /usr/share/x86_64-linux-gnu/libgmp.so.10 /usr/share/x86_64-linux-gnu/libgmp.so
> 
> This should be /usr/lib not /usr/share, but also installing libgmp-dev seems to
> work instead.
> 
> —
> Reply to this email directly or view it on GitHub.*
>